train.py

```diff
@@ -232,9 +239,15 @@
         ]
     }
     metrics["max_accuracy"] = max(accuracies)
+    closeness = np.abs(valid_loss - train_loss) + valid_loss
```

Comment:
If keeping track of closeness to train_loss to valid_loss over graph, can I ask why you are adding valid_loss to the end as well?

Reply:
Because if the train and valid loss are both close but super high it doesn't really matter. So adding the valid_loss makes that possible to track in this metric